juntagrico_billing/util/qrbill.py

```diff
@@ -84,7 +84,7 @@ def get_qrbill_svg(bill, paymenttype):
         account=stdnum.iban.compact(paymenttype.iban),
         reference_number=refnr,
         additional_information=info,
-        amount=Decimal(bill.amount_open),
+        amount=Decimal(bill.amount_open).quantize(Decimal('.01'), ROUND_HALF_UP),
```

Comment:
An easier solution would be `Decimal(str(bill.amount_open)). Would prefer that. The `quantize` solution is quite "advanced" and not very easy to understand.

Reply:
the `quantize` version is promoted to be the right one to use for monetary applications. but I added an alternative under #45

Reply:
In my opinion, using `Decimal` for business applications is overkill. We always need at most 2 decimal places. The precision of float is more than enough for that. Without using decimal the error wouldnt appear because we round the float usually when using it. We only use decimal here because the `qrbill` component expects it.
Therefore I think the usage of the `quantize` method goes further in the `decimal` direction and adds more complexity that is unnecessary. I favor the #45 solution.